### DISCUSSION TITLE
Does @vercel/og support Nodejs?

### DISCUSSION THREAD

**Discussion by moriyuu** [vercel/satori#479] [Q&A]
npmjs.com says it supports nodejs runtime, but vercel.com says it does not.

> @vercel/og supports both Node.js Runtime and Edge Runtime.
https://www.npmjs.com/package/@vercel/og 

> Use Edge Runtime by enabling the runtime: 'edge' config flag as the default Node.js runtime is not supported
https://vercel.com/docs/concepts/functions/edge-functions/og-image-generation#:~:text=Use%20Edge%20Runtime,is%20not%20supported

But according to https://github.com/vercel/next.js/issues/41968 it doesn't seem to work with nodejs runtime, so is it correct that it is not supported?
Is the documentation on npmjs.com wrong?

**Comment by theetherGit** [comment]
https://vercel.com/docs/functions/edge-functions/og-image-generation#limitations
